### PR TITLE
Add configuration parameters for static protobuf build

### PR DESCRIPTION
**Pull request by as6520**
This PR updates the protobuf build to support static build that is used for kwiver wheel.

### CODE REVIEW

**Comment by dstoup** [comment]
Jenkins test this please

**Comment by dstoup** [comment]
Jenkins test this please

File: CMake/External_Protobuf.cmake
Comment (dstoup) [inline CMake/External_Protobuf.cmake:32]: Shouldn't the C and CXX flags get appended, or is there stuff in there that will interfere?

**Comment by dstoup** [comment]
Also, those VS errors are the same 3 tests that timeout on the kwiver check. Ok to merge once the PR is approved.

File: CMake/External_Protobuf.cmake
Comment (as6520) [reply CMake/External_Protobuf.cmake:32]: Sorry didn't get the question, are these flags already added by default by fletch?

Comment (as6520) [reply CMake/External_Protobuf.cmake]: Addressed in 7255a5d

**Comment by as6520** [comment]
Jenkins test this please

**Review by dstoup — APPROVED** [review]

**Pull request merged**
